### ISSUE TITLE
Conversion References

### ISSUE THREAD

**Issue by AngHoward** [ousnius/BodySlide-and-Outfit-Studio#150]
I was following the tutorial on making a Conversion Reference and had an issue. I found a solution, but I thought it would be good to share both my issue and the solution since the solution I used was 10x easier than the one proposed in the tutorial.

So, the issue I had was with creating the "reverse" conversion reference which starts at step 6 of "the convenient way". For me, importing the slider as an obj and negating it didn't work, once the slider was in place it did absolutely nothing like I hadn't imported anything at all. I tried multiple times. Then, after getting super frustrated after at least half an hour, maybe a whole hour, of trying over and over just in case I did a step wrong, I got a hunch. Creating a conversion reference inverts the sliders and combines them into one; so maybe I could set my slider to 100% and make a conversion reference of it. That worked.

TLDR: To make a "reverse" conversion reference all you actually have to do currently is to set your original slider to 100% and make a conversion reference of it. The way you say to do it in the wiki didn't work for me and maybe that should be checked out since it could mean it may not be importing or exporting to/from objs correctly for sliders. However for now, just making a new reference with the slider at 100% works and makes the tutorial both simpler and shorter.

PS. I also had an issue where loading different presets from the slider menu messes up the body if you don't set back to zero all between presets. Basically it tries to load presets from the current preset as a baseline. Not sure if that was intended or not. Also had an issue with converting with "old" UNP armors and conversion sets since the body shape seems to vary drastically from UUNP, but that isn't your fault.

**Comment by ousnius** [comment]
Thanks, I adjusted the guide. It's indeed easier and shorter that way. When I made it, I didn't realize you could just use "make conversion reference" twice to invert it.

About loading presets in Outfit Studio, that was probably intended at some point, but it doesn't really make much sense, so I changed it in d7e2690.
